src/modules/fancyzones/lib/VirtualDesktopUtils.cpp

```diff
@@ -46,4 +49,57 @@ namespace VirtualDesktopUtils
         }
         return SUCCEEDED(CLSIDFromString(virtualDesktopId.c_str(), desktopId));
     }
+
+    bool GetCurrentVirtualDesktopId(GUID* desktopId)
```

Comment:
pass reference instead of pointer?

Reply:
I used pointer to align with other functions in `VirtualDesktopUtils` and also to win32 APIs used within these functions.